pom.xml

```diff
@@ -90,6 +90,18 @@
         <artifactId>junit-jupiter</artifactId>
         <version>${junit.jupiter.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Do we want to remove this dependency from the child pom since it's been added to the parent?